manager_test.go

```diff
@@ -258,6 +265,21 @@ func singleNotChangedStack(t *testing.T) repository {
 	}
 }
 
+func singleNotChangedStackWithEmptyModule(t *testing.T) repository {
```

Comment:
```suggestion
func singleNotChangedStackWithEmptyModuleSrc(t *testing.T) repository {
```

Or else it seems you are creating an empty module block IMHO. But not a huge deal, ignore at will.